### PR TITLE
update readme file for features available within application

### PR DESCRIPTION
**Pull request by Prem0707**
update readme file mostly in terms of features available within the application

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@
 
 # Android Client for MifosX
 
-This is an Android Application built on top of the MifosX platform for the mifos user community. This is a native Android Application written purely in Java.
+This is an Android Application built on top of the [MifosX](https://mifosforge.jira.com/wiki/spaces/MIFOSX/overview) platform and purely written in Java. It has been designed for field officers to interact with the Mifos Platform directly from the Field. 
+
+Some features that are available in the app are Search of Clients, Client Details Viewing, Savings Accounts and Loan Accounts Viewing, Savings Account Transactions (Withdrawal & Deposit), Loan Accounts (Approval, Disbursal, Repayments etc.), Identifiers and Documents (Creation, Upload, Download and View), Collection Sheet Access (Online and Offline), Datatables (View, Add/Remove Entries).
```
Comment (luckyman20) [inline README.md]: You can list the features in bullet points

Comment (Prem0707) [reply README.md]: @luckyman20 sure, I will do it by Sunday. Thanks for suggestions:)

**Comment by luckyman20** [comment]
@Prem0707 Please squash the commits

**Comment by Prem0707** [comment]
@luckyman20 Please take a look at it and suggest the necessary changes, if required. Thanks:)

File: README.md [hunk 2]
```diff
@@ -6,7 +6,15 @@
...
-This is an Android Application built on top of the MifosX platform for the mifos user community. This is a native Android Application written purely in Java.
+This is an Android Application built on top of the [MifosX](https://mifosforge.jira.com/wiki/spaces/MIFOSX/overview) platform and purely written in Java. It has been designed for field officers to interact with the Mifos Platform directly from the Field. 
+
+Some **features** that are available in the app are:
+- Search of Clients, Client Details Viewing
+- Savings Accounts and Loan Accounts Viewing
+- Savings Account Transactions (Withdrawal & Deposit)
+- Loan Accounts (Approval, Disbursal, Repayments etc.)
+- Identifiers and Documents (Creation, Upload, Download and View)
+- Collection Sheet Access (Online and Offline), Datatables (View, Add/Remove Entries).
```
Comment (tarun0) [inline README.md]: Offline collection sheet hasn't been implemented in the app yet.

**Comment by tarun0** [comment]
Fixes https://github.com/openMF/android-client/issues/1040

**Pull request merged**